SPECS/cloud-init/cloud-init.spec

```diff
@@ -18,6 +16,10 @@ Patch2:         ds-vmware-mariner.patch
 Patch3:         cloud-cfg.patch
 # Add Mariner distro support to cloud-init
 Patch4:         mariner-21.4.patch
+# backport patch https://github.com/canonical/cloud-init/commit/0988fb89be06aeb08083ce609f755509d08fa459.patch to 21.4
+Patch5:         azureds-set-ovf_is_accesible.patch
+%define python3_sitelib %{_libdir}/python3.7/site-packages
+%define cl_services cloud-config.service cloud-config.target cloud-final.service cloud-init.service cloud-init.target cloud-init-local.service
```

Comment:
This looks like the linter went a bit out of control. These defines still belong at the top of the file, and the changes below to the clean, post, and postun steps look incorrect.

The only changes in the spec should be the Release #, the Patch5 line+comment, and the changelog.

Reply:
Updated commit to just contain my changes, ignoring the spec-linter failure